src/prompt.ts

```diff
@@ -81,8 +81,9 @@ function normal(options: IPromptConfig, retries = 100): Promise<string> {
         resolve(data || options.default)
       }
     })
-    if (options.timeout) {
```

Comment:
is there a reason you removed this conditional?

Reply:
I put it in to fix this issue: https://github.com/heroku/cli/issues/880

Reply:
merge mistake, putting it back